go/vt/sqlparser/parse_test.go

```diff
@@ -800,7 +800,7 @@ func TestErrors(t *testing.T) {
 		output: "syntax error at position 25 near '::'",
 	}, {
 		input:  "update a set c = values(1)",
-		output: "syntax error at position 24 near 'values'",
+		output: "syntax error at position 26 near '1'",
```

Comment:
since we now parse the `values()` func, the problem here is the 1 now -- the parser expects a column near here